src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -132,15 +132,21 @@ export class HTMLMarkdownConverter {
 		this.turndownService.addRule('a', {
 			filter: 'a',
 			replacement: (content, node) => {
-				//On Windows, if notebook is not trusted then the href attr is removed for all non-web URL links
-				// href contains either a hyperlink or a URI-encoded absolute path. (See resolveUrls method in notebookMarkdown.ts)
-				const notebookLink = node.href ? URI.parse(node.href) : URI.file(node.title);
-				const notebookFolder = this.notebookUri ? path.join(path.dirname(this.notebookUri.fsPath), path.sep) : '';
-				let relativePath = findPathRelativeToContent(notebookFolder, notebookLink);
-				if (relativePath) {
-					return `[${node.innerText}](${relativePath})`;
+				let href = node.href;
+				const isAnchorLink = node.attributes.href.nodeValue.startsWith('#');
+				if (isAnchorLink) {
+					href = node.attributes.href.nodeValue;
```

Comment:
What's node.href at this point? I guess I would have assume that it would still be # at this point so you could just skip this and only do the else `if(!isAnchorLink)`

Reply:
node.href should contain the address but turndown parses the link as a file and instead of the original value it gets transform into `./Applications/Azure%20Data%20Studio.app/Contents/Resources/app/out/vs/code/electron-browser/workbench/workbench.html#hello`